### PR TITLE
adds `CUSTOM_TEXT` block + guide to create custom blocks 

### PR DESCRIPTION
**Pull request by dan13ram**
## Overview

**What features/fixes does this PR include?**

- Fixes existing bugs/issues with the blocks
- Adds a custom block `CUSTOM_TEXT` to the list of available blocks in dashboard / player profile / guild profile.
  - This block takes a custom title and markdown content to create a custom text block
- This PR also adds a guide which can be followed to create similar custom blocks. This guide can also be found at https://hackmd.io/@dan13ram/HJklSISio

**Please provide the GitHub issue number**

Works upon https://github.com/MetaFam/TheGame/issues/264

## Implementation

See the new guide for steps used in this PR.

### CODE REVIEW

File: packages/utils/package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "@ethersproject/address": "5.6.0",
     "bignumber.js": "^9.1.0",
     "cids": "1.1.9",
-    "ethers": "5.6.9",
+    "ethers": "5.7.0",
```
Comment (alalonde) [inline packages/utils/package.json:22]: Are the other packages using this same version ?

File: packages/web/components/Dashboard/DashboardSection.tsx [hunk 2]
```diff
@@ -46,37 +47,20 @@ const DashboardSectionInner: React.FC<Props> = ({
...
-      return 'Leaderboard';
-    case BoxTypes.DASHBOARD_COMPLETED_QUESTS:
-      return 'Submitted Quests';
-    case BoxTypes.DASHBOARD_CREATED_QUESTS:
-      return 'Posted Quest Submissions';
-    default:
-      return '';
-  }
-};
-
```
Comment (alalonde) [inline packages/web/components/Dashboard/DashboardSection.tsx:75]: Good idea refactoring this

**Review by alalonde — APPROVED** [review]
Nice! This definitely opens up a lot of customization options.

Implementation looks pretty straightforward.

File: packages/utils/package.json (hunk 1 above)
Comment (dan13ram) [reply packages/utils/package.json:22]: Yes, the `web` package was using `5.7.0` and this was creating some typescript issues for me locally.

**Pull request merged**